server/auth/index.js

```diff
@@ -2,22 +2,27 @@ const passport = require('koa-passport');
 
 const oidc = require('./oidc');
 const { STRATEGY_NAMES } = require('./constants');
```

Comment:
what is the purpose of this file? can we comment to make this choice more clear?

Reply:
would still like to comment this one